releases/release-1.31/release-notes/maps/pr-124375-map.yaml

```
@@ -0,0 +1,3 @@
+pr: 124375
+releasenote:
+  text: 'kubeadm: the global --rootfs flag is now considered non-experimental.'
```

```suggestion
  text: 'kubeadm: The global --rootfs flag considered non-experimental.'
```

https://github.com/kubernetes/sig-release/pull/2542#discussion_r1653210201